skipruntime-ts/server/src/rest.ts

```diff
@@ -97,5 +87,62 @@ export function createRESTServer(service: ServiceInstance): express.Express {
       }
     }
   });
+
+  app.put("/v1/inputs/:collection/:key", (req, res) => {
+    if (!Array.isArray(req.body)) {
+      res.status(400).json(`Bad request body ${JSON.stringify(req.body)}`);
+      return;
+    }
+    try {
+      service.update(req.params.collection, [
+        [req.params.key, req.body as Json[]],
+      ]);
+      res.sendStatus(200);
+    } catch (e: unknown) {
+      if (e instanceof UnknownCollectionError) {
+        res.sendStatus(404);
+      } else {
+        res.status(500).json(e instanceof Error ? e.message : e);
+      }
+    }
+  });
+
+  return app;
+}
+
+export function streamingService(service: ServiceInstance): express.Express {
+  const app = express();
+  app.use(express.json());
+
+  app.get("/v1/streams/:uuid", (req, res) => {
+    if (!req.accepts("text/event-stream")) {
+      res.sendStatus(406);
+      return;
+    }
+    res.writeHead(200, {
+      "Content-Type": "text/event-stream",
+      Connection: "keep-alive",
```

Comment:
no quotes, intended, why?

Reply:
Yes, IIRC the linter complains about unnecessary quotes.